CHANGELOG.md

```diff
@@ -32,7 +32,7 @@ The old code behind `peniko::Color` has been removed and color functionality is
 
 This leads to a number of breaking changes:
 
-- `peniko::Color` is now a type alias for `AlphaColor<Srgb>` from the `color` crate.
+- `peniko::Color` is gone now and what you probably want is `AlphaColor<Srgb>` from the `color` crate.
```

Comment:
I think this should mention both that:
- `AlphaColor<Srgb>` is the replacement for integration with Vello
- `Rgba8` is the like-for-like replacement

Reply:
`Rgba8` is not an appropriate replacement and isn't accepted as input for brushes or gradient color stops.

Reply:
It might be if you were using peniko outside of the linebender ecosystem.